src/features/expressions/shared-tests/functions/stringSlice/together.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Should work together with stringIndexOf",
+  "expression": [
+    "concat",
+    [
+      "stringSlice",
+      ["dataModel", "string"],
+      0,
+      ["stringIndexOf", ["dataModel", "string"], "world"]
+    ],
+    "expressions!"
+  ],
+  "expects": "Hello, expressions!",
+  "dataModel": {
+    "string": "Hello, world!"
+  }
+}
```

Comment:
This is a good example for an alternate `stringSlice` implementation. This kind of thing works only because the start is `0`. If the data model string was something like `message: Hello, world!`

This:
```json
[
  "stringSlice",
  ["dataModel", "string"],
  ["stringIndexOf", ["dataModel", "string"], "Hello"],
  ["stringIndexOf", ["dataModel", "string"], "world"]
],
```
Would result in: `Hello, world!expressions!`

To get the same result you would need to subtract the first `stringIndexOf` from the second `stringIndexOf` to get the correct length.